schema.sql

```diff
@@ -0,0 +1,10 @@
+/* Database schema to keep the structure of entire database. */
+
+ CREATE DATABASE vet_clinic;
+
+CREATE TABLE animals ( 
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(50) NOT NULL, date_of_birth DATE NOT NULL,
+  espace_attempts INT, neutered BOOLEAN NOT NULL,
+  weight_kg  DEC(11, 5)
+);
```

Comment:
- [ ] As stated in the Project requirement Kindly Add a  new column called species to your animals table. Modify your `schema.sql` file.  
-    `ALTER TABLE animals ADD species varchar(255);`